python/podio/frame.py

```diff
@@ -110,14 +110,24 @@ def __init__(self, data=None):
 
     self._param_key_types = self._get_param_keys_types()
 
+  def getAvailableCollections(self):
+    """Get the currently available collection (names) from this Frame.
+
+    Returns:
+        tuple(str): The names of the available collections from this Frame.
+    """
+    return tuple(str(s) for s in self._frame.getAvailableCollections())
+
   @property
   def collections(self):
     """Get the currently available collection (names) from this Frame.
 
     Returns:
         tuple(str): The names of the available collections from this Frame.
     """
-    return tuple(str(s) for s in self._frame.getAvailableCollections())
+    print('WARNING: collections is deprecated, use getAvailableCollections() (like in C++) instead',
+          file=sys.stderr)
```

Comment:
Can we make this a `warnings.warn("text", FutureWarning)`? That would give a bit more context as to where the warning is coming from